cate/ops/data_frame.py

```diff
@@ -230,6 +241,8 @@ def data_frame_find_closest(gdf: gpd.GeoDataFrame,
     location = GeometryLike.convert(location)
     location_point = location.representative_point()
 
+    location_point = _transform_coordinates(location_point, gdf.crs)
```

Comment:
Check if it should read:

    location_point = _transform_coordinates(location_point, gdf.crs or "EPSG:4326")